docs/RestClients.md

```diff
@@ -33,17 +33,18 @@ The `restClient` is also the ideal place to add custom HTTP headers, authenticat
 
 ## Available Clients
 
-Admin-on-rest ships 2 REST client by default:
+Admin-on-rest includes 4 REST clients by default:
 
-* simple REST: [simpleRestClient](#simple-rest) serves mostly as an example. Incidentally, it is compatible with the [FakeRest](https://github.com/marmelab/FakeRest) API.
-* **[JSON server](https://github.com/typicode/json-server)**: [jsonServerRestClient](#json-server-rest)
+* Simple REST: [marmelab/ra-data-simple-rest](https://github.com/marmelab/tree/master/packages/ra-data-simple-rest) ([read more below](#simple-rest)). It serves mostly as an example. Incidentally, it is compatible with the [FakeRest](https://github.com/marmelab/FakeRest) API.
```

Comment:
no, the `ra-` packages are for react-admin (v2), not for admin-on-rest (v1). Your changes reference ra- packages all over...

Reply:
Thanks for your review. I'm sorry my changes don't satisfy you. I will try to find some time to fix the references to react-admin. However, I'd like to point out that backporting a documentation was not the main focus of the initial issue I created. I'm neither an expert in admin-on-rest nor in react-admin and obviously can't tell the difference at this time.

Reply:
Hello Thomas, `admin-on-rest` is going to be renamed `react-admin` in the next major version. We want to emphasize that it supports more than just REST.

As your changes targets the `master` branch (hence `admin-on-rest v1), the prefix for addons is `aor`. It will become `ra` in the next major version.

Thanks :)

Reply:
Thanks for your helpful answer. I tried to revert to the old documentation document where necessary and removed the ra-* references. Hope I didn't miss anything?